### DISCUSSION TITLE
Please don't stop developing this and updating on Reddit

### DISCUSSION THREAD

**Discussion by AlexysLovesLexxie** [oobabooga/text-generation-webui#1422] [General]
Hello,
Long-time user here.

I noticed that today, you removed your Reddit presence.  This is very saddening and worrying.  I have really enjoyed using this product, and relied on your updates on Reddit to know what was new and good, and to get quick help where you could offer it.

Please, please re-consider leaving Reddit.  There are more people there who appear to be upset that you have left than you might believe.  The bulk of the community greatly appreciates your contribution.

Thank you so much for your consideration,
`AlexysLovesLexxie`

**Comment by LoopControl** [comment]
Same, Oobabooga is literally \#1 on my subreddit shortcuts and I check it multiple times a day for updates.

Please don't let people get you down, you're doing fantastic work with the project and your sharing of info.

**Comment by Anvillior** [comment]
I'll add my support to this one. Ooba is my favorite interface to use pyg with, and if they decide to stay off reddit I hope that they'll at least keep developing the github. It's thanks to Ooba that I was able to get this running locally after the Colabs started getting banned, and I can't say how grateful I am for that. Please, know that for every one jack-hole that does nothing but complain there are five of us that are beyond grateful for all the hard work you put in.

**Comment by nullgate** [comment]
What's up? I'm also on reddit for r/oobabooga only.

**Comment by poisenbery** [comment]
Reddit is a good place to chat, but from a developer perspective, it's a waste of time.
The only reason I can think of to use reddit is to ask questions, but now that we have AI that can answer faster (and without personal bias or emotional attacks) I can't really see why a talented developer would consider using that website.
Plus, Github has this discussions section now. Reddit is an outdated relic.
EDIT: A reddit user downvoted this, without offering any insight. This proves exactly what I was saying.

**Comment by oobabooga** [comment]
As a workaround, I'll try to post important new features here in the Discussions tab: https://github.com/oobabooga/text-generation-webui/discussions/categories/announcements

**Comment by AlexysLovesLexxie** [comment]
Thank you so much.  I really appreciate it.

**Comment by Malrama** [comment]
I love oobabooga so much <3
